rskj-core/src/test/java/co/rsk/peg/BridgeSupportFlyoverTest.java

```diff
@@ -1280,6 +1445,99 @@ public void registerFastBridgeBtcTransaction_funds_sent_to_retiring_fed_after_RS
         Assert.assertEquals(co.rsk.core.Coin.fromBitcoin(valueToSend).asBigInteger(), result);
     }
 
+    @Test
+    public void registerFastBridgeBtcTransaction_funds_sent_to_retiring_fed_surpasses_locking_cap_after_RSKIP293() throws BlockStoreException, BridgeIllegalArgumentException, IOException {
```

Comment:
You mean tests pre RSKIP 293 that assert funds sent to a retiring federation were not refunded?  I'll take a look I think that test may be missing. But regarding funds sent to an active federation, those are refunded properly since before RSKIP293.